pytest/tests/mocknet/mirror.py

```diff
@@ -200,14 +200,14 @@ def new_genesis_timestamp(node):
     return genesis_time
 
 
-def _apply_stateless_config(args, node):
+def _apply_stateless_config(args, node, is_validator):
     """Applies configuration changes to the node for stateless validation,
     including changing config.json file and updating TCP buffer size at OS level."""
     # TODO: it should be possible to update multiple keys in one RPC call so we dont have to make multiple round trips
     do_update_config(node, 'store.load_mem_tries_for_tracked_shards=true')
```

Comment:
RPC nodes do not need to load memtries. Explicitly set this to false if the nodes is not a validator. Doing this explicitly will make the condition resilient to changes in the default of that config.
```suggestion
    if is_validator:
        do_update_config(node, 'store.load_mem_tries_for_tracked_shards=true')
    else:
        do_update_config(node, 'store.load_mem_tries_for_tracked_shards=false')
```